### ISSUE TITLE
iOS privacy manifest

### ISSUE THREAD

**Issue by gabrielguilhoto** [mockingbot/react-native-zip-archive#302]
Starting May 1, Apple will require iOS submissions to the App Store to have privacy manifests describing the use of required reason APIs: https://developer.apple.com/news/?id=3d8a9yyh.

Could you add the required privacy manifest? `SSZipArchive` has added the manifest in version 2.5.5: https://github.com/ZipArchive/ZipArchive/issues/687. So updating `SSZipArchive` is possibly enough to solve the issue.

**Comment by MinskLeo** [comment]
Anyway, here is the patch for that, if the fix will not be applied until 1 of May by maintainer.

File:
```
patches/react-native-zip-archive+6.1.0.patch
```

Content:
```
diff --git a/node_modules/react-native-zip-archive/RNZipArchive.podspec b/node_modules/react-native-zip-archive/RNZipArchive.podspec
index 8eebcdc..14793da 100644
--- a/node_modules/react-native-zip-archive/RNZipArchive.podspec
+++ b/node_modules/react-native-zip-archive/RNZipArchive.podspec
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.preserve_paths = '*.js'
 
   s.dependency 'React-Core'
-  s.dependency 'SSZipArchive', '~>2.2'
+  s.dependency 'SSZipArchive', '~> 2.5.5'
   s.compiler_flags = '-GCC_PREPROCESSOR_DEFINITIONS="HAVE_INTTYPES_H HAVE_PKCRYPT HAVE_STDINT_H HAVE_WZAES HAVE_ZLIB MZ_ZIP_NO_SIGNING $(inherited)"'
 
   s.subspec 'Core' do |ss|

```

How to apply patch:
https://www.npmjs.com/package/patch-package

@gabrielguilhoto

**Comment by plrthink** [comment]
This has been solved by v6.1.1, thanks for @vomchik's contribution.

**Comment by balmasich** [comment]
But it require minimum deployment target 15.5
